receiver/zookeeperreceiver/scraper.go

```diff
@@ -16,31 +16,134 @@ package zookeeperreceiver
 
 import (
 	"context"
+	"errors"
+	"fmt"
+	"net"
+	"regexp"
+	"strconv"
+	"time"
 
 	"go.opentelemetry.io/collector/consumer/pdata"
+	"go.opentelemetry.io/collector/consumer/simple"
 	"go.uber.org/zap"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zookeeperreceiver/internal/metadata"
+)
+
+var zookeeperFormatRE = regexp.MustCompile(`(^zk_\w+)\s+([\w\.\-]+)`)
+
+const (
+	mntrCommand = "mntr"
```

Comment:
Should a note be added to the README that 'mntr' will need to be added to the whitelist?

`4lw.commands.whitelist=mntr`

Reply:
Added a note.